README.md

```diff
@@ -214,6 +214,28 @@ chrono.ja.parseDate('昭和６４年１月７日');
 
 Current supported locale options are: `en`, `ja`, `fr`, `nl` and `ru` (`de`, `pt`, and `zh.hant` are partially supported).
 
+#### Importing specific locales
+
+Chrono exports all locale options by default for simplicity. However, this can cause issues when using Chrono if you're using a Node.js runtime that was built with the Intl module disabled (with the `--without-intl` flag), such as: 
+```
+Invalid regular expression: /* omitted */: Invalid property name in character class
+```
+
+This is because the Intl module is required to handle special characters, such as Cyrillic (`ru`).
```

Comment:
Do you know exactly which regex or file that cause this? Could you file a separate issue for this?

Reply:
None of the regexes are actually invalid, the issue comes from instances of using Chrono with a build of Node.js that doesn't include the `Intl` module. Most static binaries, such as what `vercel/pkg` and `caoccao/Javet` use, disable the module to reduce binary size. Without the module enabled, Node.js _thinks_ that the regex is invalid because it doesn't know how to handle special characters such as Cyrillic. I don't think there is any way to "fix" this because it's not really broken, it's just the reality of using a binary without `Intl` enabled.